recipes/hyfetch/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "hyfetch" %}
+{% set version = "1.4.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/HyFetch-{{ version }}.tar.gz
+  sha256: 9fa2c9c049ebaf0ad6d4e8e076ce90e64a4b9276946a1d2ffb6912bb1a4aa327
+
+build:
+  skip: True  # [py<37]
+  entry_points:
+    - hyfetch=hyfetch.main:run
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+    - typing-extensions
```

Comment:
Is this really required for building? I cannot find it upstream.

Reply:
Removed with 181f35d. Hyfetch v1.4.10 required it to build, but upstream commit [5519dd1](https://github.com/hykilpikonna/hyfetch/commit/5519dd16a065193e85805c0e66b273fcd067993b) removed such dependency.